src/main/java/org/magicdgs/readtools/tools/distmap/DownloadDistmapResult.java

```diff
@@ -75,14 +76,23 @@
             + "Find more information about this tool in "
             + RTHelpConstants.DOCUMENTATION_PAGE + "DownloadDistmapResult.html";
 
+    // success checks variables
+    private static final String SUCCESS_FILE_NAME = "_SUCCESS";
 
     @Argument(fullName = RTStandardArguments.INPUT_LONG_NAME, shortName = RTStandardArguments.INPUT_SHORT_NAME, doc = "Input folder to look for Distmap multi-part file results. Expected to be in an HDFS file system.", common = true, optional = false)
     public String inputFolder;
 
+    private final static String PART_NAME_ARG_NAME = "partName";
     @Hidden
-    @Argument(fullName = "partName", shortName = "partName", doc = "Only download this part(s). For debugging.", optional = true)
+    @Argument(fullName = PART_NAME_ARG_NAME, shortName = PART_NAME_ARG_NAME, doc = "Only download this part(s). For debugging.", optional = true)
     public Set<String> partNames = new HashSet<>();
 
+
+    private static final String DISABLE_SUCCESS_CHECK_ARG_NAME = "disable-success-check";
+    @Advanced
+    @Argument(fullName = DISABLE_SUCCESS_CHECK_ARG_NAME, doc = "Disable the check for the _SUCCESS file on the input folder.", optional = true)
```

Comment:
@robmaz, can you comment on this argument description? It will appear as an advance argument in the help, but maybe we can be more explicit in that this is not recommended unless you know what they are doing. Any suggestion for the phrasing?